examples/sinatra/app.rb

```diff
@@ -21,6 +21,23 @@
   }
 end
 
+get '/dynamic-share' do
+  scenario = Yoti::DynamicSharingService::DynamicScenario.builder.with_policy(
+    Yoti::DynamicSharingService::DynamicPolicy.builder
+    .with_full_name
+    .with_age_over(18)
+    .with_pin_auth
+    .with_extension(Yoti::DynamicSharingService::Extension.builder.build)
```

Comment:
Shouldn't this be on the scenario rather than policy? I get `undefined method `with_extension'` error

Reply:
`Yoti::DynamicSharingService::Extension.builder.build` causes the following error when serialising `content: @content.as_json`: `undefined method 'as_json' for nil:NilClass`

Reply:
Should be fixed now? ...